ui/jobs/components/blu.js

```diff
@@ -35,6 +35,14 @@ export function setup(bars) {
     tormentBox.duration = 0;
     tormentBox.duration = 30;
   });
+  bars.onUseAbility(kAbility.AetherialSpark, () => {
+    tormentBox.duration = 0;
+    tormentBox.duration = 15 + 0.5;
+  });
+  bars.onUseAbility(kAbility.Nightbloom, () => {
+    tormentBox.duration = 0;
+    tormentBox.duration = 60 + 0.8;
```

Comment:
Where does the +0.8 and +0.5 come from?

Reply:
Animation delay.
In fact I'm thinking about use gain effect instead If this can make countdown more accurate...

Reply:
What about `onMobGainEffectFromYou` just like what you did in BRD module?

Reply:
hmmm... gaineffect is also not precise, just let it be. I'll make a comment.